crates/hotshot/src/lib.rs

```diff
@@ -545,6 +545,13 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
             &handle,
         )
         .await;
+        add_quorum_vote_task(
+            registry.clone(),
+            event_tx.clone(),
+            event_rx.activate_cloned(),
+            &handle,
+        )
+        .await;
```

Comment:
Question: is this just for testing? I don't think this is needed to run `test_quorum_vote_task` (the test should work regardless of whether we spawn the task), and this would run the task in all the examples + production.

Reply:
I need to send an update to resolve merge conflicts anyway, so will do the renaming as well!

Reply:
Oops, the above reply was meant for your other comments. 😅

Regarding this question -- I keep it there to save some work for my future integration work.

Reply:
I've commented this out as discussed on Zulip.